DigitalOffice/Services/CheckRightsService/Models/Right.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace CheckRightsService.Models
+{
+    public class Right
+    {
+        public int Id { get; set; }
```

Comment:
Why id is int? Maybe it should be guid?

Reply:
No, this Id is going to be send to front, but it has another meaning - Number from DbRight model.

Reply:
Also, about tests. 
Could you please offer some new conditionals that we have to check for these methods?